chatgprt-new proposal for documentation

shaungt1 opened the pull request:
After our discussion I cleaned up the initial README and added a docs folder where I separated content into structured files in the new docs folder. I updated all URLS to new documentation schema.

shaungt1 committed:
chatgprt-new proposal for documentation

shaungt1 committed:
update docs

lencx commented:
Thanks for the contribution, I will take some time to review it. If there are no problems, I'll merge it in the next release (early merging will cause some links to be inaccessible).

shaungt1 commented:
Moved from https://github.com/lencx/ChatGPT/issues/267

No problem glad to help, so you want only the Install, Feature, Thanks and Contributing on the initial README? and link to the official docs website in the link you provided as well?

Might be advantageous to work through or outline your ideas for each section if you want to leave notes in the pull request on what your expectations are I can go ahead and start getting them changed to meet your preferences.

Need to know know how should we address:
Which categories are staying in the "Contents" section
where you want to link or display awesome prompts instructions? 
where you want to link or display  AWESOME.md with the resources LINKS 
where you want to link or display Announcements
where you want to link or display FAQ
where to link or display Platform, Build, and Deployment Status
Features ( what content or links should go here)
where you want to link or display local development
Also which categories / sections I added we should remove

Let me know based on each of those section which should if they should be reverted back to original text ? or if we should keep the new structure per category and if we need additional details?  we might want to move all this to the PR

lencx commented:
- Keep the original installation introduction as it shows the various ways quite visually.
- `Announcement`, `MenuItem`, `Application Configuration`, `Sync Custom`, `FAQ`, `How do I build it?` moved to https://github.com/lencx/app.
- The rest are reserved for now.

But at the moment I am not completely sure about the directory structure of https://github.com/lencx/app. I've also been quite busy lately and can only devote a small amount of time to the project. I will finish the basic structure of the website as soon as possible.

shaungt1 commented:
> 

Just wanted to give an update I should have this documentation done by Friday, Id also like to get on a call in Discord to go over your ideas for the documentation on the officaal docs site so we can start organizing the documation correctly. If you can porovde a layout for what yo have in mind I doo all the work and make it happen.

lencx commented:
> Just wanted to give an update I should have this documentation done by Friday, Id also like to get on a call in Discord to go over your ideas for the documentation on the officaal docs site so we can start organizing the documation correctly. If you can provide a layout or idea for what yo have in mind in that regrads, I would be happy to get all the work done and make it happen.

The document repository is here. https://github.com/lencx/app

shaungt1 commented:
> 

Awesome, I will check it out
